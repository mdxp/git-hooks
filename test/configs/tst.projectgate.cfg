[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
	ignorecase = true
[receive]
	denyCurrentBranch = ignore
[gate]
	name = gate
	shortname = app
	notify = invalidemailaddress@abadaddress
	user-check = skip
	comment-check = skip
	review-check = skip
	mail-debug = true
[test]
	comment = I love my project gate
	files = doc/foo doc/bar appliance/server/foo
