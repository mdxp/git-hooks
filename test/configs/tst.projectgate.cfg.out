remote: changing file: appliance/server/foo[K
remote: changing file: doc/bar[K
remote: changing file: doc/foo[K
remote: Checks successful.[K
remote: invalidemailaddress@abadaddress [gate] push to app (I love my project gate)[K
remote: skipping bug update[K
